.cirrus.yml

```diff
@@ -49,7 +49,7 @@ task:
   use_compute_credits: $CIRRUS_USER_COLLABORATOR == 'true'
   name: build-ipas
   osx_instance:
-    image: high-sierra-xcode-9.4.1
+    image: catalina-flutter
   env:
     PATH: $PATH:/usr/local/bin
     matrix:
```

Comment:
Can we remove the clone of Flutter and the Flutter path additions below? According to the docs Flutter is now pre-installed on those images; not cloning should speed up jobs a little.

Reply:
I did addition cleanup, let's see if it all passes.